h/templates/help.html.jinja2

```diff
@@ -1,131 +1,223 @@
-{% extends "layouts/base.html.jinja2" %}
-
-{% set style_bundle = 'help_page_css' %}
+{% extends "layouts/base.html.jinja2" %} {% set style_bundle = 'help_page_css'
+%} {% block meta %}
+<link
+  href="//fonts.googleapis.com/css?family=Lato:400,300"
+  rel="stylesheet"
+  type="text/css"
+/>
+{{ super() }}
+<script type="application/json" class="js-hypothesis-config">
+  {
+    "openSidebar": true,
+    "openLoginForm": true
+  }
+</script>
+{% if not is_onboarding %}
+<script type="application/json" class="js-hypothesis-config">
+  {
+    "sidebarAppUrl": "{{ request.route_url('sidebar_app') }}"
+  }
+</script>
+<script async defer src="{{ embed_js_url }}"></script>
+{% endif %} {% endblock %} {% block content %}
+<div class="help-page">
+  <div class="help-page-content masthead-small">
+    {% include "includes/logo-header.html.jinja2" %}
+  </div>
+  <article class="help-page-content">
+    {% if is_help %}
+    <section class="help-page-section">
+      <h2 id="installation" class="help-page-heading">Installation</h2>
+      <p class="help-page-lede">
+        There are a couple of installations of Hypothesis to choose from:
+      </p>
+      <div class="numbered-list columns">
+        <div class="columns__col-1-2">
+          <div class="numbered-list-item">
```

Comment:
Visually the vertical alignment is not as good on this branch:

<img width="265" alt="Help branch" src="https://user-images.githubusercontent.com/2458/109186872-45310f00-7789-11eb-9a37-84b0621545ac.png">

As it is on master:

<img width="269" alt="Help master" src="https://user-images.githubusercontent.com/2458/109186859-419d8800-7789-11eb-9b2f-863e13efe422.png">

Reply:
Hah! That was a purposeful change. I'll revert.

Reply:
I dislike the way the (original) alignment looks when the content lays out on one line:

![image](https://user-images.githubusercontent.com/439947/109203879-aef1ee80-7772-11eb-8847-dd81a0fdf4b9.png)

I'll give this a little more thought.

Reply:
For reference, the updated (controversial) padding looks like this:

![image](https://user-images.githubusercontent.com/439947/109205033-0e043300-7774-11eb-8d43-4548adbec7ee.png)

and on narrower screens like this:

![image](https://user-images.githubusercontent.com/439947/109205081-1ceae580-7774-11eb-84a2-0bd973838c22.png)

Reply:
Lyza asked me to chime in here— FWIW I prefer the updated padding, where the explainer text is vertically centered with the circled numbers.